packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -22,7 +21,7 @@ import "../common/FeePayer.sol";
  * on a price feed. On construction, deploys a new ERC20, managed by this contract, that is the synthetic token.
  */
 
-contract PricelessPositionManager is FeePayer, AdministrateeInterface {
```

Comment:
nit: why was this removed? Shouldn't this administratee interface apply to all financial contracts? (this is what allows them to be emergency shutdown, etc, by the DVM)

Reply:
FeePayer is now AdminstrateeInterface so PricelessPositionManager inherits it